pytket/pytket/_tket/circuit.pyi

```diff
@@ -2572,9 +2572,9 @@ class Circuit:
         A list of all qubit ids in the circuit
         """
     @property
-    def wasm_uid(self) -> str:
+    def wasm_uid(self) -> str | None:
         """
-        :return: the wasm uid of the first wasmop found in the circuit
+        :return: the unique wasm uid of the circuit
```

Comment:
@cqc-alec this line was already changed in the other commit, but the stubs check has not shown this difference.

Reply:
Certainly it should fail if they don't match, but I'm struggling to see where there is/was a mismatch.

Reply:
The comment was changed in [14b5146](https://github.com/CQCL/tket/pull/1691/commits/14b51462f6328f618036f2e1f5297a48c3faa69a), but the CI check has not show this difference.